bats/tests/helpers/vm.bash

```diff
@@ -320,9 +325,51 @@ docker_context_exists() {
     assert_output "$RD_DOCKER_CONTEXT"
 }
 
+RD_VM_OS=
+get_os_type() {
+    [[ -n ${RD_VM_OS:-} ]] || load_var RD_VM_OS || true
+    if [[ -z ${RD_VM_OS:-} ]]; then
+        local id is_rd=false
+        for id in $(rdctl shell cat /etc/os-release | tr -d '"' | awk -F= '/^ID/ { print $2 }'); do
+            case "$id" in
+            opensuse | suse) RD_VM_OS=opensuse ;;
+            alpine) RD_VM_OS=alpine ;;
+            rancher-desktop-wsl-distro) is_rd=true ;;
+            esac
+        done
+        if [[ -z ${RD_VM_OS:-} ]] && $is_rd; then
+            RD_VM_OS=alpine # The existing Alpine-based distribution
+        fi
+        save_var RD_VM_OS
+    fi
+    if [[ -z ${RD_VM_OS:-} ]]; then
+        fail "Failed to detect OS"
+    fi
+    echo "${RD_VM_OS}"
+}
+
+service_control() { # service action
+    local if_started=
```

Comment:
Maybe drop the trailing `=`. It looks like there is something missing. We later use it as `${if_started:-}` anyways. Otherwise use `""` to initialize to the empty string, if you think that is better.